### ISSUE TITLE
Skypecalling bot issue

### ISSUE THREAD

**Issue by umangsp** [Microsoft/BotBuilder-Samples#286]
hi I am facing issue in skypecalling bot. I am successfully able to connect to bot. It greets me and ask for dialpad input. however It never recognize my voice. the return text is always blank.

application ID is .

e58765a4-6c8e-4f0d-b5ba-b1544b1ce238

Please help me.

### SDK Platform

.net SDK C#

### URL of Code Sample
I think i have issue in config file.
https://github.com/Microsoft/BotBuilder-Samples/blob/master/CSharp/skype-CallingBot/Web.config
### Problem Description

hi I am facing issue in skypecalling bot. I am successfully able to connect to bot. It greets me and ask for dialpad input. however It never recognize my voice. the return text is always blank.

application ID is .

e58765a4-6c8e-4f0d-b5ba-b1544b1ce238

-->

**Comment by stevkan** [comment]
I'm looking into this and will reach out to you.

**Comment by umangsp** [comment]
Hi Stevkan,
Thanks for looking into it.

 I am finally able to resolve this with so much permutation computation. 
here is what I did last.

I changed MSCognitiveservice.cs file from 

//var requestUri = this.speechRecognitionUri + Guid.NewGuid();

To hardcoded value of.

            var requestUri = @"https://speech.platform.bing.com/recognize?scenarios=smd&appid=D4D52672-91D7-4C74-8AD8-42B1D98141A5&locale=en-US&device.os=bot&form=BCSSTT&version=3.0&format=json&instanceid=565D69FF-E928-4B7E-87DA-9A750B96D9E3&requestid=" + Guid.NewGuid();

Want to know this will only work with Bing service ? how to utilize speech api preview instead. 

Any help on configurations will help.

**Comment by stevkan** [comment]
Hi @umangsp,

Again, apologies for the delay. Look at [this repo](https://github.com/stevkan/skype-calling-bot) of the Skype calling bot. It's a working copy that I made some short time ago.

I made a change in the web.config to use this url in appSettings. You will need to create an account and provide the associated Api key:
```cs
<add key="MicrosoftSpeechApiKey" value="<API KEY>" />    
<add key="MicrosoftSpeechRecognitionUri" value="https://speech.platform.bing.com/speech/recognition/dictation/cognitiveservices/v1?language=en-US" />
```
In MSCognitiveSpeechService.cs, line 34:
```cs
var requestUri = this.speechRecognitionUri; // + Guid.NewGuid();
```
And, line 52:
```cs
return data["DisplayText"];
```

Start with those changes and see if they make a difference. I made a few other changes however those were for other tests I was performing and shouldn't have an effect on your success either way. But, let me know.
